examples/connector/java/build.gradle

```diff
@@ -68,7 +68,10 @@ tasks.register('deleteBuild', Delete) {
 tasks.register('copyProtos', Copy) {
     from file("$rootDir/../../..")
     into file("src/main/proto/")
-    include "*.proto"
+    include "*v2.proto"
+    rename { String fileName ->
```

Comment:
What is this renaming for? We are not going to rename these new files, they will always have the version number in them

Reply:
We are renaming while copying to example folder.

Reply:
This is done to ensure we don't need to update import statements to v2 - https://github.com/fivetran/fivetran_sdk/pull/47/files#diff-9d2d8a9c2db1bfd763e54f465fbf48bdf54cc587c459ce96c0ebbe3e78ceff98R7